sympy/core/tests/test_subs.py

```diff
@@ -831,3 +831,12 @@ def test_issue_6976():
     assert (x**4 + x**3 + x**2 + x + sqrt(x)).subs(x**2, y) == \
         y**Rational(1, 4) + y**Rational(3, 2) + sqrt(y) + y**2 + y
     assert x.subs(x**3, y) == y**Rational(1, 3)
+
+
+def test_issue_11746():
+    x, y = symbols('x y')
```

Comment:
I think these symbols are defined at the top of the file so that they don't need to be defined in each test

Reply:
Sure, I made the change.  Thanks for the review!